model_draft_transactions.go

```diff
@@ -104,6 +104,25 @@ func getDraftTransactionID(ctx context.Context, xPubID, id string,
 	return draftTransaction, nil
 }
 
+// getDraftTransactions will get all the draft transactions with the given conditions
+func getDraftTransactions(ctx context.Context, metadata *Metadata, conditions *map[string]interface{},
+	queryParams *datastore.QueryParams, opts ...ModelOps) ([]*DraftTransaction, error) {
+
+	modelItems := make([]*DraftTransaction, 0)
+	if err := getModelsByConditions(ctx, ModelDraftTransaction, &modelItems, metadata, conditions, queryParams, opts); err != nil {
+		return nil, err
```

Comment:
Does opts need the `...`

`opts` vs `opts...`

Not sure if that is an error or not...